sdk/onlineexperimentation/azure-onlineexperimentation/azure/onlineexperimentation/models/_models.py

```diff
@@ -0,0 +1,527 @@
+# pylint: disable=line-too-long,useless-suppression
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=useless-super-delegation
+
+import datetime
+from typing import Any, Dict, List, Literal, Mapping, Optional, TYPE_CHECKING, Union, overload
+
+from .. import _model_base
+from .._model_base import rest_discriminator, rest_field
+from ._enums import ExperimentMetricType
+
+if TYPE_CHECKING:
+    from .. import models as _models
+
+
+class AggregatedValue(_model_base.Model):
+    """An event property value aggregated by a metric.
+
+    :ivar event_name: The name of the event. Required.
+    :vartype event_name: str
+    :ivar filter: [Optional] A condition to filter events.
```

Comment:
Are there known values for filter?

Reply:
This is a KQL expression string, values will depends on the customer's event schema and metric definition. The samples and documentation cover these in substantial detail, probably too much to cover in parameter docs.